Sources/Classes/core/JavascriptBridge/MiniAppJavaScriptEnumerations.swift

```diff
@@ -129,6 +130,7 @@ public enum MiniAppEvent: String {
     case resume = "miniappresume"
     case secureStorageReady = "miniappsecurestorageready"
     case secureStorageError = "miniappsecurestorageloaderror"
+    case miniappRecieveJsonString = "miniappreceivejsoninfo"
```

Comment:
```suggestion
    case miniappReceiveJsonString = "miniappreceivejsoninfo"
```

Reply:
This is now updated in the latest commit please review.